addons/project/views/project_sharing_views.xml

```diff
@@ -38,9 +38,8 @@
                 <field name="is_closed"/>
                 <field name="displayed_image_id"/>
                 <field name="active"/>
-                <field name="allow_subtasks"/>
-                <field name="child_text"/>
-                <field name="is_closed"/>
```

Comment:
Make them invisible instead of removing them. 🙂

Reply:
Done